inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -649,12 +653,19 @@ public void onRecommendationRejectEvent(AjaxRecommendationRejectedEvent aEvent)
     public void onRecommendationAcceptEvent(AjaxRecommendationAcceptedEvent aEvent)
     {
         AnnotatorState annotatorState = ActiveLearningSidebar.this.getModelObject();
-        predictionModel = recommendationService.getPredictions(annotatorState.getUser(),
+        model = recommendationService.getPredictions(annotatorState.getUser(),
             annotatorState.getProject());
         AnnotatorState eventState = aEvent.getAnnotatorState();
-        AnnotationObject acceptedRecommendation = predictionModel
-                .getPredictionByVID(aEvent.getVid());
+        SourceDocument document = annotatorState.getDocument();
+        VID vid = aEvent.getVid();
+        Optional<AnnotationObject> oRecommendation = model.getPredictionByVID(document, vid);
 
+        if (!oRecommendation.isPresent()) {
+            LOG.error("Could not find annotation in [{}] with id [{}]", document, vid);
```

Comment:
Such a problem should also be logged to the user. We are usually in an interactive cycle when hitting this method.